helpers/python.ts

```diff
@@ -155,18 +157,20 @@ const getAdditionalDependencies = (
       });
       break;
     case "openai":
-      dependencies.push({
-        name: "llama-index-llms-openai",
-        version: "^0.2.0",
-      });
-      dependencies.push({
-        name: "llama-index-embeddings-openai",
-        version: "^0.2.3",
-      });
-      dependencies.push({
-        name: "llama-index-agent-openai",
-        version: "^0.3.0",
-      });
+      if (templateType !== "multiagent") {
```

Comment:
No OpenAI package needed for multi agent?

Reply:
It has OpenAI packages in pyproject.toml already